credential.go

```diff
@@ -122,6 +147,20 @@ func (ic *Credential) CreateDisclosureProofBuilder(disclosedAttributes []int, no
 		d.attrRandomizers[v], _ = common.RandomBigInt(ic.Pk.Params.LmCommit)
 	}
 
+	if rangeStatements != nil {
+		d.rpStructures = make(map[int][]*rangeproof.ProofStructure)
+		for index, statements := range rangeStatements {
+			if !isUndisclosedAttribute(disclosedAttributes, index) {
+				return nil, errors.New("Range statements on revealed attributes are not supported")
+			}
+			d.rpStructures[index] = nil
```

Comment:
Why is this line here? As far as I can tell it is is a no-op; `d.rpStructures[index]` will already be nil.

Reply:
fixed